src/components/Flex.tsx

```diff
@@ -0,0 +1,17 @@
+import styled from '@emotion/styled';
+import { CSSProperties } from 'react';
+
+interface FlexProps {
+  align?: CSSProperties['alignItems'];
+  justify?: CSSProperties['justifyContent'];
+  direction?: CSSProperties['flexDirection'];
+}
+
+const Flex = styled.div<FlexProps>(({ align, justify, direction }) => ({
+  display: 'flex',
+  alignItems: align,
+  justifyContent: justify,
+  flexDirection: direction,
+}));
```

Comment:
💬 태그드 템플릿으로 하지 않은 이유가 있나용?!

```suggestion
const Flex = styled.div<FlexProps>`
  display: flex;
  align-items: ${({ align }) => align};
  justify-content: ${({ justify }) => justify};
  flex-direction: ${({ direction }) => direction};
`;
```

Reply:
flex에서 gap도 많이 쓰이는 속성인데 추가되면 좋을 것 같네요!

Reply:
```tsx
${({ align }) => align};
${({ justify }) => justify};
..
```
위처럼 하나하나 props를 꺼내오는 과정이 귀찮아서 화살표 함수로 작성했었습니다!
일관성을 위해서 태그된 템플릿으로 작성하는 것이 좋겠네요!